### ISSUE TITLE
INPUT/OUTPUT works, but nor FORWARDING

### ISSUE THREAD

**Issue by actuallymentor** [ilikenwf/pg2ipset#5]
Hello,

First off, thanks for your great work. This is really very useful :)

I find that everything works great, except for the FORWARDING iptables rules. E.g.:

When I ping a blacklisted IP from inside the server it is blocked
When connected to the server as a VPN I can ping the blacklisted IP just fine

I've tried to debug this on my own but would love your opinion.

Thanks!

**Comment by actuallymentor** [comment]
Update, UFW seemed to be interfering. I uninstalled it. Now I can;t ping blacklisted IPs from within the server, but when connected through a VPN connection there is no internet connectivity at all.

Suggestions would be highly appreciated.

**Comment by jokogr** [comment]
Could you post your current iptables rules?

Just run `iptables -nvL` and `iptables -t nat -nvL` with sufficient privileges to do so.

**Comment by actuallymentor** [comment]
Thanks very much for the response.

iptables -nvL gives: http://pastebin.com/jTWgCZCt
iptables -t nat -nvL: http://pastebin.com/DJ0NjaDp

**Comment by jokogr** [comment]
I think you have pasted the same information twice and anyway, it appears that there are still UFW chains which take precedence over the rules ipset-update.sh makes.

Just flush your chains and start over.

**Comment by actuallymentor** [comment]
Thanks for your response.

I must have messed the copy/paste, apologies.

I tried in a simplified manner: uninstalled ufw and used forward only. This also does not work: http://pastebin.com/206Kwqis

Thank you so much for taking some time to help figure this out.

**Comment by actuallymentor** [comment]
Upadate.

I started adding lists manually. The issue starts when the Bogon set is added. I'll dive into it now.

Pastebin of the list in question: http://pastebin.com/VNLnrvJQ

**Comment by ilikenwf** [comment]
Strange - please let me know what you find out. It could be as simple as an errant character somewhere.

**Comment by actuallymentor** [comment]
@ilikenwf I was not able to solve the issue. Have spent another couple of hours at it but my brain couldn't handle seeing any more IPs.

Basically now I isolated the Bogon list as an issue. Once its ipset rule is activated, all traffic dies. I have tried searching the list, but grep doesn't return any matches for 8.8 (I used 8.8.8.8 as one of the test ping IPs).

If there is any tests I can do so we can isolate the issue together please let me know.

**Comment by elico** [comment]
@actuallymentor are you still trying to figure this issue out?

**Comment by actuallymentor** [comment]
@elico no, I remember having this issue but abandoned this tool for my particular use case.
